### DISCUSSION TITLE
Un-hide columns

### DISCUSSION THREAD

**Discussion by mylenrath** [RafaelGB/obsidian-db-folder#780] [Q&A] [answered]
I want to temporarily hide certain DB columns, but then un-hide them later when needed.  Is there a way to revert a hidden column without recreating the entire database?

**Comment by RafaelGB** [comment, marked as answer]
yeah, there is an entire columns menu clicking the add column button (top right)

**Reply by emign** [reply]
The problem is, that when you hide the file name it wont show up again.
